tests/constants-e2e.ts

```diff
@@ -1,7 +1,18 @@
-import { Address, Token } from '../src/types';
+import {
+  SmartTokenParams,
+  balanceOfFn,
+  allowanceFn,
+  SmartToken,
+} from '../tests/smart-tokens';
+import { Address } from '../src/types';
 import { ETHER_ADDRESS, Network } from '../src/constants';
 
-export const Tokens: { [network: number]: { [symbol: string]: Token } } = {
+export const GENERIC_ADDR1 = '0xbe9317f6711e2da074fe1f168fd9c402bc0a9d1b';
```

Comment:
Where is this used? I can not find

Reply:
https://github.com/paraswap/paraswap-dex-lib/blob/b26673fbe227a16049d2db3bc14f9a900a155d8c/src/dex/generic-rfq/generic-rfq-e2e.test.ts#L47